openc3-cosmos-cmd-tlm-api/app/controllers/storage_controller.rb

```diff
@@ -193,9 +193,9 @@ def delete_bucket_item(params)
     raise "Unknown bucket #{params[:bucket]}" unless bucket_name
     path = sanitize_path(params[:object_id])
     key_split = path.split('/')
-    # Anywhere other than config/SCOPE/targets_modified requires admin
-    if !(params[:bucket] == 'OPENC3_CONFIG_BUCKET' && key_split[1] == 'targets_modified')
-      return unless authorization('admin')
+    # Anywhere other than config/SCOPE/targets_modified or config/SCOPE/tmp requires admin
+    if !(params[:bucket] == 'OPENC3_CONFIG_BUCKET' && (key_split[1] == 'targets_modified' || key_split[1] == 'tmp'))
+      return false unless authorization('admin')
```

Comment:
If return value matters then it should be explicit at the end of the method.

Reply:
We're short cutting this method because we don't want to delete unless they are admin